src/pages/tutorial/angular/step-3.mdx

```diff
@@ -0,0 +1,455 @@
+---
+title: 3. Using APIs
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### This step takes our static components and populates them with data from the GitHub GraphQL API – loading states and all. We'll be displaying Carbon repository information in a data table.
+
+<AnchorLinks>
+
+<AnchorLink>Fork, clone and branch</AnchorLink>
+<AnchorLink>Install dependencies</AnchorLink>
+<AnchorLink>Create access token</AnchorLink>
+<AnchorLink>Connect to Apollo</AnchorLink>
+<AnchorLink>Fetch data</AnchorLink>
+<AnchorLink>Populate data table</AnchorLink>
+<AnchorLink>Add loading</AnchorLink>
+<AnchorLink>Add pagination</AnchorLink>
+<AnchorLink>Submit pull request</AnchorLink>
+
+</AnchorLinks>
+
+### Preview
+
+The [GitHub GraphQL API](https://developer.github.com/v4/) is very well documented, and even though the focus of this tutorial isn't learning and using GraphQL, it's a great opportunity to fetch Carbon-related data for this Carbon tutorial.
+
+To do so, we'll be using Apollo Client, the front-end component of the [Apollo Platform](https://www.apollographql.com/docs/intro/platform). Apollo provides several open source tools for using GraphQL throughout your application’s stack. Apollo Client is a sophisticated GraphQL client that manages data and state in an application.
+
+A [preview](https://angular-step-4-carbon-tutorial.netlify.com) of what you will build (see repositories page):
+
+<Preview
+  height="400"
+  title="Carbon Tutorial Step 3"
+  src="https://angular-step-4-carbon-tutorial.netlify.com"
```

Comment:
![image](https://user-images.githubusercontent.com/1691245/67426896-a1d25780-f5a0-11e9-9a4d-9aba56aa17eb.png)

* Should Angular be deployed in development mode?
* Maybe use a local image for the favicon instead of carbondesignsystem.com?
* Issue with GitHub access token?

Reply:
> Should Angular be deployed in development mode?

Probably not ... the netlify build should use the `--prod` flag for the CLI

Reply:
fixed